packages/ruleset/src/utils/get-schema-type.js

```diff
@@ -10,22 +10,22 @@ const checkCompositeSchemaForConstraint = require('./check-composite-schema-for-
  * to a specific language type or data structure in an SDK).
  */
 const SchemaType = {
-  ARRAY: Symbol(),
-  BINARY: Symbol(),
-  BOOLEAN: Symbol(),
-  BYTE: Symbol(),
-  DATE: Symbol(),
-  DATE_TIME: Symbol(),
-  DOUBLE: Symbol(),
-  ENUMERATION: Symbol(),
-  FLOAT: Symbol(),
-  INT32: Symbol(),
-  INT64: Symbol(),
-  INTEGER: Symbol(),
-  NUMBER: Symbol(),
-  OBJECT: Symbol(),
-  STRING: Symbol(),
-  UNKNOWN: Symbol()
+  ARRAY: Symbol('array'),
```

Comment:
these changes ensure that the specified unique string appears in the toString() output